src/main/java/com/alipay/oceanbase/rpc/util/MonitorUtil.java

```diff
@@ -117,7 +117,10 @@ private static String logMessage(String traceId, String database, String tableNa
             endpoint = endpoint.replaceAll(",", "#");
         }
 
-        String argsValue = buildParamsString(Arrays.asList(rowKeys));
+        String argsValue = "null";
```

Comment:
If _rowKeys_ is empty, will the _argsValue_ equal to ""? Maybe you could use `isEmpty()` to find out whether it is "".

Reply:
If you want to print rowkey in log, I guess you should explicit what is null. Make it clear.

Reply:
already edit